landscaper/pkg/gardenlet/controller/gardenlet_delete.go

```diff
@@ -49,11 +49,11 @@ func (g Landscaper) Delete(ctx context.Context) error {
 		if isSeedUsedByAnyShoot(g.gardenletConfiguration.SeedConfig.Name, shootList.Items) {
 			return fmt.Errorf("cannot delete seed '%s' which is still used by at least one shoot", g.gardenletConfiguration.SeedConfig.Name)
 		}
+	}
 
-		err := g.deleteSeedAndWait(ctx, seed)
-		if err != nil {
-			return err
-		}
+	err = g.deleteSeedAndWait(ctx, seed)
+	if err != nil {
```

Comment:
Why this change?/why not this:

```suggestion
	if err := g.deleteSeedAndWait(ctx, seed); err != nil {
```

Reply:
Reverted the change in error. thanks for spotting.